Samples~/SampleGame/Assets/Scripts/Server/GameLiftServer.cs

```diff
@@ -64,29 +86,19 @@ public void TerminateGameSession(bool processEnding)
 
         try
         {
-            GenericOutcome outcome = GameLiftServerAPI.TerminateGameSession();
```

Comment:
TerminateGameSession has been deprecated in the Server SDK 5.x, so we should remove this whole method and replace with ProcessEnding calls only